auction/src/lib.rs

```diff
@@ -33,13 +28,11 @@ decl_event!(
 	}
 );
 
-type AuctionIdLinkedItem<T> = LinkedItem<<T as Trait>::AuctionId>;
-
 decl_storage! {
 	trait Store for Module<T: Trait> as Auction {
 		pub Auctions get(fn auctions): map hasher(twox_64_concat) T::AuctionId => Option<AuctionInfo<T::AccountId, T::Balance, T::BlockNumber>>;
 		pub AuctionsIndex get(fn auctions_index): T::AuctionId;
-		pub AuctionEndTime get(fn auction_end_time): map hasher(twox_64_concat) (T::BlockNumber, Option<T::AuctionId>) => Option<AuctionIdLinkedItem<T>>;
+		pub AuctionEndTime get(fn auction_end_time): double_map hasher(twox_64_concat) T::BlockNumber, hasher(twox_64_concat) T::AuctionId => Option<T::AuctionId>;
```

Comment:
```suggestion
		pub AuctionEndTime get(fn auction_end_time): double_map hasher(twox_64_concat) T::BlockNumber, hasher(twox_64_concat) T::AuctionId => Option<()>;
```

It is now possible to iterate keys and values

Reply:
@xlc cool. didn't know that

Reply:
it is new feature is substrate alpha 5, which we just upgraded yesterday